CI issue with precompile

I have an issue with precompilation from ReferenceTests since today here: https://github.com/Wikunia/ConstraintSolver.jl/runs/2030426740?check_suite_focus=true

the error message looks like this:
```
ERROR: LoadError: AssertionError: precompile(Tuple{typeof(test_reference), File{DataFormat{:PNG}}, Matrix{RGB{Float64}}, Function, Nothing})
Stacktrace:
 [1] top-level scope at /home/runner/.julia/packages/ReferenceTests/44qin/src/ReferenceTests.jl:27
``` 

**Update:**
This also happens locally in a new temporary environment just for reference tests:
```
➜ julia --project=$(mktemp -d)                                                                                                                                                                                                              
               _                                                                                                                                                                                                                            
   _       _ _(_)_     |  Documentation: https://docs.julialang.org                                                                                                                                                                         
  (_)     | (_) (_)    |                                                                                                                                                                                                                    
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.                                                                                                                                                                             
  | | | | | | |/ _` |  |                                                                                                                                                                                                                    
  | | |_| | | | (_| |  |  Version 1.5.3 (2020-11-09)                                                                                                                                                                                        
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release                                                                                                                                                                           
|__/                   |                                                                                                                                                                                                                    
                                                                                                                                                                                                                                            
(tmp.FasdasQads) pkg> add ReferenceTests
...
julia> using ReferenceTests
[ Info: Precompiling ReferenceTests [324d217c-45ce-50fc-942e-d289b448e8cf]
ERROR: LoadError: AssertionError: precompile(Tuple{typeof(test_reference), File{DataFormat{:PNG}}, Matrix{RGB{Float64}}, Function, Nothing})
Stacktrace:
 [1] top-level scope at /home/ole/.julia/packages/ReferenceTests/44qin/src/ReferenceTests.jl:27
 [2] include(::Function, ::Module, ::String) at ./Base.jl:380
 [3] include(::Module, ::String) at ./Base.jl:368
 [4] top-level scope at none:2
 [5] eval at ./boot.jl:331 [inlined]
 [6] eval(::Expr) at ./client.jl:467
 [7] top-level scope at ./none:3
in expression starting at /home/ole/.julia/packages/ReferenceTests/44qin/src/ReferenceTests.jl:24
ERROR: Failed to precompile ReferenceTests [324d217c-45ce-50fc-942e-d289b448e8cf] to /home/ole/.julia/compiled/v1.5/ReferenceTests/Rkqd1_jPXuE.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1305
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923
```

Any idea why this is happening?

--- a/Project.toml
+++ b/Project.toml
@@ -1,7 +1,7 @@
 name = "ReferenceTests"
 uuid = "324d217c-45ce-50fc-942e-d289b448e8cf"
 authors = ["Christof Stocker <stocker.christof@gmail.com>", "Lyndon White <oxinabox@ucc.asn.au>", "Johnny Chen <johnnychen94@hotmail.com>"]
-version = "0.9.4"
+version = "0.9.5"
 
 [deps]
 DeepDiffs = "ab62b9b5-e342-54a8-a765-a90f495de1a6"

--- a/src/ReferenceTests.jl
+++ b/src/ReferenceTests.jl
@@ -22,10 +22,15 @@ include("equality_metrics.jl")
 include("render.jl")
 
 if Base.VERSION >= v"1.4.2" && ccall(:jl_generating_output, Cint, ()) == 1
-    @assert precompile(Tuple{Core.kwftype(typeof(test_reference)),NamedTuple{(:by,), Tuple{typeof(isequal)}},typeof(test_reference),String,Matrix{RGB{Float64}}})   # time: 0.4697069
-    @assert precompile(Tuple{Core.kwftype(typeof(test_reference)),NamedTuple{(:by,), Tuple{typeof(isequal)}},typeof(test_reference),String,Matrix{RGBA{Float64}}})   # time: 0.24671553
-    @assert precompile(Tuple{typeof(test_reference),File{DataFormat{:PNG}},Matrix{RGB{Float64}},Function,Nothing})   # time: 0.029915236
-    @assert precompile(Tuple{typeof(test_reference),File{DataFormat{:PNG}},Matrix{RGBA{Float64}},Function,Nothing})   # time: 0.013356352
+    precompile(Tuple{Core.kwftype(typeof(test_reference)),NamedTuple{(:by,), Tuple{typeof(isequal)}},typeof(test_reference),String,Matrix{RGB{Float64}}})  || @warn "Failure to precompile ReferenceTests.test_reference(String,Matrix{RGB})"
+    precompile(Tuple{Core.kwftype(typeof(test_reference)),NamedTuple{(:by,), Tuple{typeof(isequal)}},typeof(test_reference),String,Matrix{RGBA{Float64}}}) || @warn "Failure to precompile ReferenceTests.test_reference(String,Matrix{RGBA})"
+    if isdefined(FileIO, :action)
+        precompile(Tuple{typeof(test_reference),File{DataFormat{:PNG},String},Matrix{RGB{Float64}},Function,Nothing})  || @warn "Failure to precompile ReferenceTests.test_reference(File{format\"PNG\",String},Matrix{RGB})"
+        precompile(Tuple{typeof(test_reference),File{DataFormat{:PNG},String},Matrix{RGBA{Float64}},Function,Nothing}) || @warn "Failure to precompile ReferenceTests.test_reference(File{format\"PNG\",String},Matrix{RGBA})"
+    else
+        precompile(Tuple{typeof(test_reference),File{DataFormat{:PNG}},Matrix{RGB{Float64}},Function,Nothing})  || @warn "Failure to precompile ReferenceTests.test_reference(File{format\"PNG\"},Matrix{RGB})"
+        precompile(Tuple{typeof(test_reference),File{DataFormat{:PNG}},Matrix{RGBA{Float64}},Function,Nothing}) || @warn "Failure to precompile ReferenceTests.test_reference(File{format\"PNG\"},Matrix{RGBA})"
+    end
 end
 
 end # module

